.github/workflows/close-stale.yaml

```diff
@@ -0,0 +1,27 @@
+name: Close Stale PRs
+on:
+  schedule:
+    - cron: 0 6 * * *
+jobs:
+  stale:
+    name: Closing stale PRs
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-issue-stale: -1
+          days-before-pr-stale: 20
+          stale-pr-message: >-
+            Hi,
+
+
+            This PR has not seen activity in 20 days. Therefore, we are marking the PR as stale for now. It will be closed after 7 days.
+
+            If you need help with the PR, do not hesitate to reach out in the winglang community slack at [winglang.slack.com](https://winglang.slack.com).
+
+            Feel free to re-open this PR when it is still relevant and ready to be worked on again.
+
+            Thanks!
```

Comment:
Is an extra newline being added for each line?

Reply:
I want to say this won't have extra newlines since value in the YAML file starts with `>-` instead of `>`,  but let's see what happens